gateway/api.go

```diff
@@ -2652,6 +2656,26 @@ func (gw *Gateway) getOauthClientDetails(keyName, apiID string) (interface{}, in
 	return reportableClientData, http.StatusOK
 }
 
+func (gw *Gateway) oAuthTokensHandler(w http.ResponseWriter, r *http.Request) {
+	if !r.URL.Query().Has("scope") {
+		doJSONWrite(w, http.StatusUnprocessableEntity, apiError("scope parameter is required"))
+		return
+	}
+
+	if r.URL.Query().Get("scope") != "lapsed" {
```

Comment:
if there is no other choice, why not `lapsed` bool? Or do we plan to add other values for scope?

Reply:
we had this discussion, we started with a boolean first and changed it to scope considering the potential of adding different values such as `expired`, `all`